app/assets/stylesheets/application.css.scss

```diff
@@ -1,10 +1,6 @@
-@import 'variables/app';
-@import 'required';
-@import 'identity-style-guide/dist/assets/scss/packages/required';
-@import 'design-system-waiting-room';
-@import 'identity-style-guide/dist/assets/scss/packages/global';
-@import 'identity-style-guide/dist/assets/scss/packages/components';
-@import 'components/all';
-@import 'identity-style-guide/dist/assets/scss/packages/utilities';
-@import 'utilities/all';
-@import 'print';
+@forward 'required';
+@forward 'uswds';
```

Comment:
To do: Try out the new component cherry-picking optimization, both (a) to reduce stylesheet size and (b) to make sure it works as expected.